src/components/Doc/ComponentDoc.js

```diff
@@ -49,6 +50,21 @@ export const ComponentDoc = ({
         <title>{pageTitle || name}</title>
         <meta name="description" content={description} />
       </Helmet>
+      {stable && (
+        <Notification
+          background="light-3"
+          status="info"
+          global
+          actions={[
+            {
+              label: 'stable branch.',
+              href: '/stablebranch',
+              target: '_blank',
+            },
+          ]}
+          message={`New! ${name} is in beta, test it out and let us know what you think. This component is available on the`}
```

Comment:
The text is feeling crowded to be by the containing Box. Playing around with padding to give it more room to breathe, then adding some corner rounding to take the hard edges off and be more harmonious with the button style and interactive code container.

Also, I threw in a bit more spacing between the notification and the page title -- the title is getting squeezed. Open to whether the spacing is applied as a gap on a bounding Box or as margin on the Notification.

```suggestion
          message={`New! ${name} is in beta, test it out and let us know what you think. This component is available on the`}
          round="xsmall"
          pad={{ horizontal="medium", vertical: "small" }}
```

**Before**
![Screen Shot 2022-09-12 at 9 30 11 AM](https://user-images.githubusercontent.com/1756948/189694712-22017b2a-4f61-42bc-9f03-626a67f0b916.png)

**After**
![Screen Shot 2022-09-12 at 9 29 58 AM](https://user-images.githubusercontent.com/1756948/189694749-411b30ac-63c8-4aa8-90fd-399993cd709b.png)

Reply:
I'd recommend addressing any styling issues with the base grommet theme, if we're feeling that all notifications would benefit, especially with the text padding aspect here, and probably the rounding.

Reply:
Looks alot better thanks @halocline for your changes

Reply:
Changes look good! 

> I'd recommend addressing any styling issues with the base grommet theme, if we're feeling that all notifications would benefit, especially with the text padding aspect here, and probably the rounding.

Wouldn't a change like this cause backwards compatibility issues with the theme?

Reply:
If we feel that the "grommet" theme seems a bit "broken", we can change it. It would be the same if did something like change a color to improve accessibility.

Reply:
I pushed up changes to add a little more spacing between the notification and page title. For the other changes, rounding and extra padding within the notification would it make sense to open up an issue on grommet for this so we can explore if we want to make this change in the theme?

Reply:
sure, or just go ahead and change the theme

Reply:
Agree, make change in Grommet theme. New functionality for Notification introduced, Grommet theme had not yet embraced / adopted that new functionality until now. Changes to grommet theme would be justified.

Reply:
Sounds good! I will do that